Sources/Core/Shared/Operation.swift

```diff
@@ -379,11 +379,6 @@ public extension Operation {
             }
         })
 
-        // The evaluator waits for all condition dependencies
-        conditions
```

Comment:
Yeah, still working on this. Might just draw a diagram to explain the scheduling.